travel-event-host-app/src/components/searchSection/searchSection.tsx

```diff
@@ -0,0 +1,80 @@
+'use client';
+import styles from './styles.module.css';
+
+import FilterBox from './searchEventsFilterBox/filterBox';
+import { useState, useEffect } from 'react';
+import { Box, MenuItem, Select, Typography } from '@mui/material';
+import EventCard from '../event/event-card/Event-card';
+import Event from '@/models/event';
+import CategoriesType from '@/components/searchSection/types';
+import { useRouter } from 'next/navigation';
+import { SearchInput } from './searchInput/searchInput';
+import { getEventsBySearchQuery } from '@/app/clients/event/event-client';
+
+export default function SearchSection({ keyword }: { keyword: string }) {
+  const [sortBy, setSortBy] = useState<string>('Date');
+  const [resultEventList, setResultEventList] = useState<Event[]>([]);
+  const [categories, setCategories] = useState<CategoriesType>({});
```

Comment:
Let's rename this to better reflect what this is. It looks like it's keeping track of the state of the categories checkboxes. 
Suggestion: `categoryCheckboxState`, `setCategoryCheckboxState`.